libs/wingsdk/package.json

```diff
@@ -140,7 +140,7 @@
     "@types/aws-lambda",
     "cdktf",
     "cron-parser",
-    "esbuild-wasm",
+    "esbuild",
```

Comment:
I think bundledDeps still run postInstall scripts, let's see!

Reply:
Ok so they do run, but don't seem to like the fact that they have already run before when initially bundled. We do a different approach